### DISCUSSION TITLE
How can I access the core files after installing on DigitalOcean?

### DISCUSSION THREAD

**Discussion by rescue168** [requarks/wiki#3234] [Help / Questions] [answered]
I host my Wiki on DigitalOcean, and I’d like to gain access to the main html files to edit some things. Is there a way to do that?

**Comment by NGPixel** [comment, marked as answer]
If you installed the Wiki.js image from the DigitalOcean marketplace, then you simply need to go inside the docker container. From the terminal, run:
```
docker exec -it wiki /bin/sh
```

You're now in the container. The app is found under `/wiki`
